.travis.yml

```diff
@@ -5,9 +5,20 @@ node_js:
   - '10'
 install:
   - npm i npminstall && npminstall
+before_script:
+  - sudo echo 'sentinel monitor mymaster 127.0.0.1 6379 2
+sentinel down-after-milliseconds mymaster 60000
+sentinel failover-timeout mymaster 180000
+sentinel parallel-syncs mymaster 1' > sentinel.conf
+  - sudo echo 'sentinel monitor mymaster 127.0.0.1 6379 2
+sentinel down-after-milliseconds mymaster 60000
+sentinel failover-timeout mymaster 180000
+sentinel parallel-syncs mymaster 1
+port 26380' > sentinel.conf_1
+  - sudo redis-server /etc/redis/redis.conf
+  - sudo redis-server sentinel.conf --sentinel
+  - sudo redis-server sentinel.conf_1 --sentinel
```

Comment:
This file is auto-generated by egg-ci, it's not supposed to be modified manually.